README.md

```diff
@@ -54,11 +54,11 @@ To compile the LuaRock modules:
 3. Run `luarocks make` (replace the version number as necessary):
     ```bash
     # Base SDK
-    luarocks make launchdarkly-server-sdk-1.0-0.rockspec \
+    luarocks make launchdarkly-server-sdk-2.0.0-0.rockspec \
```

Comment:
Could you wrap these with:
```
<!-- x-release-please-start-version -->

<!-- x-release-please-end -->
```

We do this in a few readme files.

Reply:
I guess you have this in your version updater thingie.

Reply:
Yep. I originally didn't do it because I thought the x-release-please updater was going to use `v`-prefixed version (don't know why I had that impression 🤷‍♂️ )

I would like to replace it, but I'll do it in a different PR.